package.json

```diff
@@ -6,150 +6,153 @@
+    "history": "^4.7.2",
+    "i18next": "^11.3.6",
+    "i18next-browser-languagedetector": "^2.2.2",
+    "lodash": "^4.17.10",
+    "moment": "^2.22.2",
+    "moment-timezone": "^0.5.21",
+    "normalize.css": "^8.0.0",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-fontawesome": "^1.6.1",
     "react-helmet": "^5.2.0",
     "react-highlighter": "^0.4.2",
-    "react-i18next": "~7.5.1",
-    "react-onclickoutside": "~6.7.1",
-    "react-redux": "~5.0.7",
-    "react-spinkit": "~3.0.0",
-    "react-styled-flexboxgrid": "^2.2.0",
-    "react-tooltip": "^3.4.1",
-    "recompose": "^0.26.0",
-    "redux": "~3.7.2",
-    "redux-actions": "~2.3.0",
-    "redux-first-router": "^0.0.16-next",
+    "react-i18next": "^7.8.1",
+    "react-onclickoutside": "^6.7.1",
+    "react-redux": "^5.0.7",
+    "react-spinkit": "^3.0.0",
+    "react-styled-flexboxgrid": "^2.4.0",
+    "react-tooltip": "^3.6.1",
+    "recompose": "^0.27.1",
+    "redux": "^4.0.0",
+    "redux-actions": "^2.4.0",
+    "redux-first-router": "^0.0.20-next",
     "redux-first-router-link": "^1.4.2",
-    "redux-logger": "~3.0.6",
-    "redux-responsive": "^4.3.6",
-    "redux-thunk": "~2.2.0",
-    "styled-components": "^3.2.5",
-    "url-pattern": "^1.0.3",
-    "webfontloader": "~1.6.28",
-    "whatwg-fetch": "~2.0.4"
+    "redux-logger": "^3.0.6",
+    "redux-responsive": "^4.3.7",
+    "redux-thunk": "^2.3.0",
+    "styled-components": "^3.3.3",
+    "whatwg-fetch": "^2.0.4"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.44",
-    "@babel/core": "^7.0.0-beta.44",
-    "@babel/plugin-transform-react-jsx": "^7.0.0-beta.44",
-    "@babel/plugin-transform-runtime": "~7.0.0-beta.44",
-    "@babel/preset-env": "~7.0.0-beta.44",
-    "@babel/preset-flow": "^7.0.0-beta.44",
-    "@babel/preset-react": "~7.0.0-beta.44",
-    "@babel/preset-stage-1": "~7.0.0-beta.44",
-    "@babel/runtime": "~7.0.0-beta.44",
-    "@integreat-app/eslint-config": "~0.1.5",
-    "@integreat-app/eslint-config-jest": "~0.1.1",
-    "@integreat-app/eslint-config-react": "~0.1.3",
-    "assets-webpack-plugin": "~3.5.1",
-    "autoprefixer": "~8.2.0",
+    "@babel/cli": "^7.0.0-beta.54",
+    "@babel/core": "^7.0.0-beta.54",
+    "@babel/plugin-transform-react-jsx": "^7.0.0-beta.54",
+    "@babel/plugin-transform-runtime": "^7.0.0-beta.54",
+    "@babel/preset-env": "^7.0.0-beta.54",
+    "@babel/preset-flow": "^7.0.0-beta.54",
+    "@babel/preset-react": "^7.0.0-beta.54",
+    "@babel/preset-stage-1": "^7.0.0-beta.54",
+    "@babel/runtime": "^7.0.0-beta.54",
+    "@integreat-app/eslint-config": "^0.2.1",
+    "@integreat-app/eslint-config-jest": "^0.2.1",
+    "@integreat-app/eslint-config-react": "^0.2.1",
+    "assets-webpack-plugin": "^3.8.4",
+    "autoprefixer": "^9.0.0",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "~8.2.3",
-    "babel-jest": "^22.4.3",
-    "babel-loader": "~8.0.0-beta.2",
+    "babel-eslint": "^8.2.6",
+    "babel-jest": "22.4.4",
```

Comment:
Why are you missing the caret ^ here?

Reply:
Because we need version 22 because of an incompatibility with the IDE

Reply:
Can we add a comment somewhere so we don't forget to upgrade it when the issue is resolved? (With link to the GitHub issue)

Reply:
Also the caret only corresponds to minor package upgrades like 22.4.x. So this wouldn't upgrade jest to v23 or similar :thinking:

Reply:
Do you want to do it? I don't know what exactly went wrong 🙈

Reply:
done